tests/model/test_model.py

```diff
@@ -0,0 +1,59 @@
+from platform import platform
+from typing import Any
+
+import pytest
+
+from zulipterminal.model import Model
+
+
+class TestModel:
+    @pytest.fixture(autouse=True)
+    def mock_external_classes(self, mocker: Any) -> None:
+        self.controller = mocker.patch('zulipterminal.core.'
+                                       'Controller',
+                                       return_value=None)
+
+    @pytest.fixture
+    def model(self, mocker, initial_data, user_profile):
+        mocker.patch('zulipterminal.model.Model.get_messages')
+        mocker.patch('zulipterminal.model.Model.fetch_initial_data',
+                     return_value=initial_data)
+        mocker.patch('zulipterminal.model.Model.get_all_users',
+                     return_value=[])
+        mocker.patch('zulipterminal.model.Model.get_subscribed_streams',
+                     return_value=[])
+        # NOTE: PATCH WHERE USED NOT WHERE DEFINED
+        self.classify_unread_counts = mocker.patch(
+            'zulipterminal.model.classify_unread_counts',
+            return_value=[])
+        mocker.patch('zulipterminal.core.Controller.client.get_profile',
+                     return_value=user_profile)
+        return Model(self.controller)
+
+    def test_init(self, model, initial_data, user_profile):
```

Comment:
I feel like there's not much value in this test, given all the other test we are running.

Reply:
but it's recommended to have one, right?